sys-process/audit/audit-3.0.6.ebuild

```diff
@@ -144,14 +140,19 @@ multilib_src_install_all() {
 	# newinitd "${FILESDIR}"/auditd-init.d-2.4.3 auditd
 	# newconfd "${FILESDIR}"/auditd-conf.d-2.1.3 auditd
 
+	# Flatcar: install sample configuration
+	insinto /usr/share/auditd
+	doins "${FILESDIR}"/init.d/auditd.conf
+	newtmpfiles "${FILESDIR}/auditd-conf.tmpfiles" auditd-conf.conf
```

Comment:
```suggestion

```
Since you added it to the rules tmpfile this here is redundant (and doesn't work because the file does not exist)